integration-test/src/test/java/com/datastrato/gravitino/integration/test/util/spark/SparkUtilIT.java

```diff
@@ -78,8 +83,14 @@ protected List<Object[]> sql(String query) {
     return rowsToJava(rows);
   }
 
+  // It's hard to parse Spark SQL output, create SparkTableInfo from SparkBaseTable.
   protected SparkTableInfo getTableInfo(String tableName) {
-    return SparkTableInfo.getSparkTableInfo(sql("DESC TABLE EXTENDED " + tableName));
+    Dataset ds = getSparkSession().sql("DESC TABLE EXTENDED " + tableName);
+    CommandResult result = (CommandResult) ds.logicalPlan();
+    DescribeRelation relation = (DescribeRelation) result.commandLogicalPlan();
+    ResolvedTable table = (ResolvedTable) relation.child();
+    SparkBaseTable baseTable = (SparkBaseTable) table.table();
```

Comment:
Can you confirm that the force type conversion was successful?

Reply:
For current spark version, I think it's YES, For other spark versions may need some adaptor work.